homeassistant/components/number/const.py

```diff
@@ -86,6 +86,12 @@ class NumberDeviceClass(StrEnum):
 
     # NumberDeviceClass should be aligned with SensorDeviceClass
 
+    AMMONIA = "ammonia"
```

Comment:
This PR is changing our entity model. There needs to be approval in a discussion in our architecture repository first.

https://github.com/home-assistant/architecture/discussions

https://developers.home-assistant.io/docs/core/entity#changing-the-entity-model